pkgs/development/python-modules/dbus-python-client-gen/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, into-dbus-python
+, dbus-python
+, psutil
+, python-dateutil
+, wcwidth
+}:
+
+buildPythonPackage rec {
+  pname = "dbus-python-client-gen";
+  version = "0.8";
+
+  src = fetchFromGitHub {
+    owner = "stratis-storage";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-nSzxT65WHBVct5pGHmIAHJXftd0tKZeK/argN+V9xcs=";
+  };
+
+  propagatedBuildInputs = [
+    into-dbus-python
+    dbus-python
+  ];
+
```

Comment:
It would be good to enforce testing by adding `ceckInputs = [ pytestCheckHook ]` and `pythonImprotsCheck = [ "..." ];` if possible

Reply:
to add on this 

Would you please use `pytestCheckHook` instead? This is a bit more nix friendly, and also ensures that tests are run as part of a build.

More information can be found on the [pytestCheckHook](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook) section of the manual.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.